ytfzf

```diff
@@ -802,13 +829,60 @@ scrape_pt () {
 		thumbs: .thumbnailUrl
 	    }
 	]')
-     export videos_json
+     printf "%s" "$videos_json" > "$tmp_video_json_file"
      videos_data="$(get_video_data "$videos_json")"
      function_exists "on_video_data_gotten" && on_video_data_gotten "$videos_data" "$videos_json" "$pt_json"
      [ $show_thumbnails -eq 1 ] && download_thumbnails "$videos_json"
+     wait
      unset pt_json
 }
 
+scrape_odysee () {
+    odysee_json=$(
+	curl \
+	    -s "https://lighthouse.lbry.com/search" \
+	    -G --data-urlencode "s=$*" \
+	    -G --data-urlencode "mediaType=video,audio" \
+	    -G --data-urlencode "include=channel,title,thumbnail_url,duration,cq_created_at,description" \
+	    -G --data-urlencode "size=$odysee_video_search_count" \
+	    -G --data-urlencode "time_filter="$(printf "$upload_date_filter" | tr -d '-')"" \
```

Comment:
curl `-G` indicates a get request, it suffices to put it once.

The rest seems great